device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D24C52/sai_3800_24x50g_52x100g.xml

```diff
@@ -0,0 +1,470 @@
+<?xml version="1.0"?>
+<root>
+    <platform_info type="3800">
+
+        <!-- Device MAC address  -->
+        <device-mac-address>00:02:03:04:05:00</device-mac-address>
+
+		<!-- ISSU enabled -->
+		<issu-enabled>1</issu-enabled>
```

Comment:
@madhanmellanox please fix alignment. Also make sure spaces (not tabs) are used.

Reply:
sai_3800_24x50g_52x100g.xml is a copy of ../ACS-3800/sai_3800.xml